src/PrestaShopBundle/Form/Admin/Type/TranslatableType.php

```diff
@@ -116,6 +122,31 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
+        $errors = [];
+
+        foreach ($view->vars['errors'] as $existingError) {
+            $errors[] = $existingError;
```

Comment:
based on https://github.com/symfony/symfony/blob/v3.4.43/src/Symfony/Component/Form/FormView.php
index `errors` can be undefined in `$view->vars` array

If it's defined and is an array, why not `$errors = $view->vars['errors']`

So can you try `$errors = $view->vars['errors'] ?? [];`

Reply:
This makes sense, thanks!

Reply:
Ahh never mind. $view->vars['errors'] is an iterator not an array. That's why we have

Reply:
you can use ̀iterator_to_array`or try to cast it to array ?

Reply:
iterator_to_array to array worked perfectly, thanks!